Update interfaces/pointerevents.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/99d8a4a/ed/idl/pointerevents.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/610489327

foolip reviewed (changes requested):
Oops, actually this regresses some tests, those need updating.

autofoolip committed:
Update interfaces/pointerevents.idl

Source: https://github.com/tidoust/reffy-reports/blob/99d8a4a/ed/idl/pointerevents.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/610489327

foolip commented:
This is blocked on https://chromium-review.googlesource.com/c/chromium/src/+/1906903 landing.
